FindTarget.py

```diff
@@ -442,17 +442,52 @@ def findTape(contours, image, centerX, centerY, mask, CornerMethod):
         # Sort contours by area size (biggest to smallest)
         cntsSorted = sorted(contours, key=lambda x: cv2.contourArea(x), reverse=True)[:1]
        
-        for cnt in cntsSorted:
-            xb, yb, wb, hb = cv2.boundingRect(cnt)
```

Comment:
Aren't these variables used later in the code in Brian's method?   I think by removing this code, you will break Brian's two methods because they rely on xb, yb, wb, or hb.

Reply:
Confirmed.   Line 512 further down in the code uses these variables:  
 ROI_mask = mask[yb:yb+hb, xb:xb+wb]

If we merged this the code would fail if we tried to run method 8.   Please make sure you test all the methods to ensure they will continue to work.

Reply:
I fixed Brian's variables and tested methods 8 and 9.